dotnetv3/cross-services/Anniversary_SNS/README.md

```diff
@@ -0,0 +1,70 @@
+# Using the Amazon API Gateway to invoke Lambda Functions
+
+This example shows you how to create an AWS Lambda function that runs
+on a defined frequency using Amazon CloudWatch sheculed events. When the
+application runs, the AWS Lambda function checks the DynamoDB table to
+find employees who are having their one-year anniversary.
+
+Those having a work anniversary will receive a text message, sent using
+Amazon Simple Notification Service (Amazon SNS).
+
+## Solution Components
+
+| Solution | Description |
+|----------|-------------|
+| CreateTable | A console application that creates a DynamoDB table called EmployeeTable. Edit the code to give at least one of the emplloyees a valid phone mobile phone number for testing purposes. |
+| DeleteTable | A console application that deletes the table created for this example. Run it when you are done to remove the table you created. |
+| SendSnsMessage | An AWS Lambda function that checks for employees having a work anniversary and send an anniversary message to the employee's phone. |
+
+## Prerequisites
+
+- An AWS Account with proper credentials.
+- A C# .NET IDE
+- AWS Toolkit for .NET
+- .NET Core 3.1 or later
+
+The example also uses a Dynamo DB table that contains the following information:
+
+| ID | FirstName | Phone | HireDate |
+|----|-----------|-------|----------|
+| 101 | Jadwiga | 11234567890 | current date - one year |
+| 102 | Denis | 11234567890 | 2017-3-9 |
+| 103 | Sean | 11234567890 | 2013-5-13 |
+
+## Setting up the environment
+
+There are several steps you will need to perform in order to use the example
+Lambda function included with this code example:
```

Comment:
verbose; suggest 
Perform the following steps to use the example Lambda function: